src/metricscollector/appmetricprune/prune/prune.go

```diff
@@ -0,0 +1,33 @@
+package prune
```

Comment:
The naming isn't good. We don't need it to live so far down.
prune.go should just like in the metricscollector and call the struct Pruner.

You will then wire this in as an ifrit.Process that runs 1,2 or 4 times a day. I don't really care on the interval. We can make that configurable too.